File: 1024.py
test, fake = int(input())


for index in range(1, test+1):
    inventory = []

    for number in range(101):
        inventory.append(0)
    
    for many in range(1000):
        grade = int(input())
        inventory[grade] += 1
    
    maximum = int(max(inventory))
    maximum_number = 0
    
    for number in range(1, 101):
        if maximum == inventory[number]:
            maximum_number = number
    
    print(f"#{index} {maximum_number}")

File: 1024.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

void grade(){
    int test_number, number, max = 0, maximum_number;
    int list[101] = {0};

    cin >> test_number;

    for(int i = 0; i < 1000; i++){
        cin >> number;
        list[number]++;
    }

    for(int i = 1; i <= 100; i++){
        if(list[i] >= max){
            maximum_number = i;
            max = list[i];
        }
    }

    cout << "#" << test_number << " " << maximum_number << endl;
}

int main(){
    int test, number, fake;

    cin >> test;

    for(int i = 0; i < test; i++) grade();

    return 0;
}